modules/nw-service-externalip-create.adoc

```diff
@@ -6,70 +6,109 @@
+. Attach an ExternalIP resource to the service by completing the following steps:
++
+.. Optional: If you are creating a new service, specify a value in the `spec.externalIPs` field and array of one or more valid IP addresses in the `allowedCIDRs` parameter.
 +
+.Example of service YAML configuration file that supports an ExternalIP resource
 [source,yaml]
 ----
 apiVersion: v1
 kind: Service
 metadata:
   name: svc-with-externalip
 spec:
-  ...
   externalIPs:
-  - 192.174.120.10
+    policy:
+      allowedCIDRs:
+      - 192.168.123.24/30
 ----
-
-.. If you are attaching an ExternalIP to an existing service, enter the following command. Replace `<name>` with the service name. Replace `<ip_address>` with a valid ExternalIP address. You can provide multiple IP addresses separated by commas.
 +
-[source,terminal]
+.. Set the allowed CIDR IP address blocks for your cluster:
++
+[source,terminal,subs="attributes+"]
 ----
-$ oc patch svc <name> -p \
-  '{
-    "spec": {
-      "externalIPs": [ "<ip_address>" ]
-    }
-  }'
+$ oc patch svc <service_name> -n <project_external_ip> -p ' <1>
+{"spec":{"externalIP":{"policy":{"allowedCIDRs":["192.168.123.24/30"]}}}}'
+  --type='merge'
 ----
 +
-For example:
+Where:
++
+`<service_name>` is the service name and `<project_external_ip>` is *X*. For the remaining steps, `service-example` replaces `<service_name>`.
++
+.. Validate the updated ExternalIP resource configuration:
 +
 [source,terminal]
 ----
-$ oc patch svc mysql-55-rhel7 -p '{"spec":{"externalIPs":["192.174.120.10"]}}'
+$ oc get networks.config cluster -o jsonpath='{.spec.externalIP}{"\n"}'
+{"policy":{"allowedCIDRs":["192.168.123.24/30"]}}'
+----
++
+.. Attach the ExternalIP resource to the service:
++
+.Example that attaches the ExternalIP resource to the `service-example` service
+[source,terminal,subs="attributes+"]
+----
+$ oc patch svc/service-example -n <project_external_ip> -p '{"spec":{"externalIPs":["192.168.123.25"]}}’
+----
++
+.. Confirm that an external IP address attached to the service:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get svc
 ----
 +
 .Example output
 [source,terminal]
 ----
-"mysql-55-rhel7" patched
+NAME              TYPE        CLUSTER-IP     EXTERNAL-IP      PORT(S)             AGE
+service-example ClusterIP   172.30.5.225   192.168.123.25   8080/TCP,8888/TCP   24h
 ----
-
-. To confirm that an ExternalIP address is attached to the service, enter the following command. If you specified an ExternalIP for a new service, you must create the service first.
++
+.. Check the ExternalIP resource status:
 +
 [source,terminal]
 ----
-$ oc get svc
+$ curl 192.168.123.25:8080
 ----
 +
 .Example output
 [source,terminal]
 ----
-NAME               CLUSTER-IP      EXTERNAL-IP     PORT(S)    AGE
-mysql-55-rhel7     172.30.131.89   192.174.120.10  3306/TCP   13m
+Hello OpenShift!
 ----
```

Comment:
Lines 42 through 114 I don't think are an improvement.  It's just my implementation specific notes.  I think this part of the doc is okay as-is.

Reply:
Reverted